src/Tasks/GenerateResource.cs

```diff
@@ -2432,6 +2432,12 @@ internal Assembly ResolveAssembly(object sender, ResolveEventArgs args)
                     {
                         if (String.Compare(requestedAssemblyName.Name, candidateAssemblyName.Name, StringComparison.CurrentCultureIgnoreCase) == 0)
                         {
+                            string itemSpec = _assemblyFiles[i].ItemSpec;
+
+                            if (requestedAssemblyName.Name.Contains("System.Runtime"))
```

Comment:
Interesting. I'm not tied to `ReflectionOnlyLoad`; it's just closest to what was done before. Since this didn't make 15.3, I'll explore `System.Reflection.Metadata`.